### PR TITLE
got the media_type from selected item

### PR DESCRIPTION
**Pull request by fatshotty**
fix for skin widgets.
Got the media_type from selected item instead of Container

### CODE REVIEW

**Comment by razzeee** [comment]
Can you give more context, what exactly this fixes?

**Comment by fatshotty** [comment]
Hi,
you can reproduce the problem using a skin that is based on skin-shortcuts, such as ArcticHorizon, Bello, Aura... 
To reproduce:
- Switch skin to ArcticHorizon or Aura (for example)
- Configure custom widget for homepage
- add widget "recently added movies" in homepage
- navigate on a movie item and open Context Menu
- Select "Rate on Trakt" (for example)
- nothing happen

Issue is related to the function `kodiUtilities.getMediaInfo()` because it is based on `'Container.Content(movies)` , but in the Home page widget, there is no `content`. 
`Container.Content(movies)` returns 'None' .
My fix make it based on `ListItem.DBTYPE` (the selected ListItem)


Kodi_Media_Center
Kodi_Media_Center

**Comment by razzeee** [comment]
Thank you much clearer, will have a look this evening

File: resources/lib/kodiUtilities.py [hunk 1]
```diff
@@ -367,14 +367,28 @@ def checkAndConfigureProxy():
 
 
 def getMediaType():
-    if xbmc.getCondVisibility('Container.Content(tvshows)'):
+    # if xbmc.getCondVisibility('Container.Content(tvshows)'):
```
Comment (razzeee) [inline resources/lib/kodiUtilities.py]: Can you get rid of the commented out part?

**Comment by fatshotty** [comment]
yeah sure :)

**Comment by fatshotty** [comment]
it has been done on my `main` branch.
should I redo the "pull - request" ?

**Comment by razzeee** [comment]
Only if it's a problem for you. I'm hoping to test and merge this in the coming days.

**Comment by fatshotty** [comment]
many thanks ;)

**Comment by fatshotty** [comment]
good.
is the addon already upgradable on my kodi? :)

**Comment by razzeee** [comment]
No, I would normally batch some more changes into one release, and the last release was just 6 days ago. So I'll wait at least another week, before shipping a new build.

**Pull request merged**